bin/ch/ch.cpp

```diff
@@ -769,6 +769,11 @@ int _cdecl wmain(int argc, __in_ecount(argc) LPWSTR argv[])
         ChakraRTInterface::UnloadChakraDll(chakraLibrary);
     }
 
+    if (ttUri != nullptr)
+    {
+        free(ttUri);
```

Comment:
I've also used Utf8.Detach and then free in couple of places. However on our discussion with @curtisman heard that Windows DLL allocation map might differ then the calling process. We should introduce a free mechanism for Utf8 or use new JSStringFree here. IIRC `.so` shouldn't have a problem with it.